upgrade/upgrade-scripts/common.sh

```diff
@@ -431,6 +431,18 @@ function fix_and_migrate_services() {
 		fi
 	fi
 
+	#
+	# The services listed below are permanently disabled.  We need to
```

Comment:
I think this works for the use case we have, but still misses one; which is a new service in the category of "disabled by default but could be enabled dynamically". If we ever have a service like that, neither this code block nor the one below it would correctly mask it. I think to handle all four cases:
1. A new service that is unconditionally disabled
2. A new service that is disabled by default but dynamically enabled
3. An existing service that is unconditionally disabled
4. An existing service that is disabled by default but dynamically enabled

... we could change the `is_svc_masked_or_isabled` function to return true if the service doesn't exist at all (yet), right? Then, we wouldn't need two lists of services.